tolopay/src/Components/Sidebar/styles.js

```diff
@@ -0,0 +1,149 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+import { btnReset, v } from "../../Utils/variables";
+
+export const SSidebar = styled.div`
+    margin-top: -85px;
+    z-index: 1000;
+     width: ${({ isOpen }) => (!isOpen ? `auto` : v.sidebarWidth)};
+   
+    background: ${({ theme }) => theme.bg};
+    height: 100vh;
```

Comment:
nice for using 100vh.

but theme switcher is not visible without scrolling.
![layout view v1](https://user-images.githubusercontent.com/30285329/190852377-fc1d6404-f0de-4b82-9e12-96a2aa122b87.png)